src/components/DoneTasksList/DoneTasksList.jsx

```diff
@@ -0,0 +1,63 @@
+import './DoneTasksList.css';
+import TaskListItem from "../TaskListItem/TaskListItem";
+import { useState, useEffect, useCallback } from "react";
+import { useSelector } from 'react-redux';
+import { CSSTransition } from 'react-transition-group';
+import { Box, IconButton, Typography } from "@mui/material";
+import KeyboardArrowRightOutlinedIcon from '@mui/icons-material/KeyboardArrowRightOutlined';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+
+export default function DoneTasksList({ tasksArray }) {
+    const [visibilityOfDoneTasks, setVisibilityOfDoneTasks] = useState(false);
+    const [doneTasksCount, setDoneTasksCount] = useState(null)
+    const tasks = useSelector((state) => state.tasks.tasks);
+
+    const countDoneTasks = useCallback(() => {
+        const doneTasksArray = tasksArray.filter(task => task.done === true)
+        setDoneTasksCount(doneTasksArray.length);
+    }, [tasksArray]);
+
+    useEffect(() => {
+        countDoneTasks()
+    }, [tasks, countDoneTasks]);
+
+    return (
+        <>
+            <Box
+                onClick={() => { setVisibilityOfDoneTasks(!visibilityOfDoneTasks) }}
```

Comment:
```suggestion
                onClick={() => setVisibilityOfDoneTasks(!visibilityOfDoneTasks)}
```
plese create a separate function next time

Reply:
done